Resources/Prototypes/Maps/game.yml

```diff
@@ -41,6 +41,10 @@
     prefixCreator: 'VG'
   mapPath: Maps/packedstation.yml
   minPlayers: 15
+  conditions:
+    - !type:HolidayMapCondition
+      inverted: true
+      holidays: [ "Christmas" ]
   overflowJobs:
```

Comment:
?

Reply:
Dunno where that came from, but it's most definitely NOT my code. I didn't write that.
Removed it.